samples/scala-replicatedentity-examples/src/test/scala/com/example/replicated/counter/SomeCounterIntegrationSpec.scala

```diff
@@ -41,6 +41,16 @@ class SomeCounterIntegrationSpec extends AnyWordSpec with Matchers with BeforeAn
       getResult.futureValue.value shouldBe (42 - 32)
     }
 
+    "Test value after passivation" in {
+      val counterId = "1"
+
+      counter.increase(IncreaseValue(counterId, 1)).futureValue
+      Thread.sleep(5000)
+
+      val getResult = counter.get(GetValue(counterId))
+      getResult.futureValue.value shouldBe 1
+    }
```

Comment:
I just wanted to have a test somewhere confirming behaviour. I'm not sure we should add one of these for each entity, or just wait for the proper TCK tests to be updated.

Reply:
Just this for now seems good, remove again once TCK tests added.

Reply:
tks, added a comment to remove after tck tests added.